recipes/flex/all/test_package/basic_nr.cpp

```diff
@@ -0,0 +1,1699 @@
+#line 2 "test_package/basic_nr.cpp"
```

Comment:
It seems that this is the flex generated output and this file should not be commited.

Reply:
ok, it should be better now, thanks !